src/types.ts

```diff
@@ -48,4 +49,16 @@ export type SsoProfileConfig = {
   sso_region: string,
   sso_role_name: string,
   sso_start_url: string,
+  sso_session?: string,
 };
+
+export type ConfigData = {
+  config: IniFileContent;
+  keys: string[];
+  values: Record<string, string>[];
+};
+
+export type SsoIniLoader =
+   IniLoader & {
+     loadSsoSessionsFrom(options: LoadFileOptions): IniFileContent;
```

Comment:
It won't be necessary anymore when https://github.com/aws/aws-sdk-js/pull/4456 is merged.